libs/gson-support/pom.xml

```diff
@@ -46,6 +47,24 @@
     </dependencies>
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>
+                            ${project.groupId}.${project.artifactId}
```

Comment:
✏️  Probably doesn't matter but might want to use a delimiter other than `.` or `-` because many group IDs and artifact IDs already are dot-delimited (third party / servicemix) or dash-delimited (internal bundles). 

Had to say something. Likely not worth changing though since this is only one bundle we're talking about.

Reply:
I copied that from other modules in libs and thirdparty. So at least it has consistency going for it.